app/src/main/java/com/example/campusguide/search/PopupSearchLocationListener.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.campusguide.search
+
+import androidx.appcompat.app.AppCompatActivity
+import com.example.campusguide.Constants
+import com.example.campusguide.directions.DirectionsFlow
+import com.example.campusguide.map.GoogleMapAdapter
+import com.example.campusguide.search.travelWindow.TravelWindow
+import com.example.campusguide.search.travelWindow.TravelWindowClickListener
+import com.example.campusguide.search.travelWindow.TravelWindowData
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.Marker
```

Comment:
this line will have to be removed, in my PR i made a new marker interface we have to use

Reply:
i merged my branch, def remove this line ! and use the Marker in campusguide

Reply:
I will do that!